### DISCUSSION TITLE
howto remove instaces that failed during creation process

### DISCUSSION THREAD

**Discussion by intelicomap** [eldy/SellYourSaas#153] [Q&A]
We ran out of space in the home directory, when the deploy script is creating the home directory, it fails and we need to clean unwanted files. What is the best approach to do this?

**Comment by karozans** [comment]
Did you ever figure out how to do this?  This forum does not seem to be monitored by anyone and the devs seem to ignore all requests and bugs.  The Dolibarr forums are very active, but for some reason they ignore the sellyoursaas module.  I think they are the same people.

**Reply by eldy** [reply]
You can also force the status of contract to "deploy" and the click on the undeploy to correctly clean. If it fails, set status to une deploy manually and try to run the clean sh script.

**Comment by hregis** [comment]
@karozans Hello, you have a script in "/sellyoursaas/scripts/clean.sh"
launch this script with "test" and "confirm" parameters in your instance server...

/home/admin/wwwroot/dolibarr_sellyoursaas/scripts/clean.sh test
/home/admin/wwwroot/dolibarr_sellyoursaas/scripts/clean.sh confirm

**Reply by karozans** [reply]
Awesome, thank you.  I actually achieved it by tracking down the information in the dbase and manually removed it.

The script is much better.  Thanks
